web/src/test/java/com/epam/rft/atsy/cucumber/util/CommonStepDefs.java

```diff
@@ -16,4 +27,73 @@ public void loggedIn() {
     getDriver().findElement(By.id("password")).sendKeys("pass3");
     waitForPageLoadAfter(event -> getDriver().findElement(By.id("loginButton")).click());
   }
+
+  @When("^the user clicks on (.*) in the header$")
+  public void theUserClicksOnButtonInTheHeader(String button) throws Throwable {
+      switch (button) {
```

Comment:
We will eliminate the `switch` statements later, when introducing the Page Object pattern.